src/Domain/Model/Movie/MovieBuilder.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Fusani\Movies\Domain\Model\Movie;
+
+class MovieBuilder
```

Comment:
This may be overkill but I didn't want to repeat the conditionals below in multiple places in the repository.